test.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
-var assert = require('assert');
-var arrayDifference = require('./');
+import test from 'ava';
+import arrayDifference from './';
 
-it('should filter out the difference', function () {
-	assert.deepEqual(arrayDifference([2, 3, 4], [3, 50, 60]), [2, 4]);
+test('should filter out the difference', t => {
```

Comment:
followed the existing style, again, not sure about the improvement of the suggested change. I would defer it to @sindresorhus 

I have been following relying on `xo` for the coding convention.